spec/unit/inflecto/inflections/plural_spec.rb

```diff
@@ -24,7 +24,7 @@
     before { object.uncountable(rule) }
 
     it 'removes it from uncountables' do
-      subject.uncountables.should eq([])
+      subject.uncountables.should be_empty
```

Comment:
I also prefer equal > eql > eq strategy, but usually for collections i use less restrictive matchers to have less reason to change specs. But you are right: in this case eql is better